paper/main.tex

```diff
@@ -440,20 +439,20 @@ \subsection*{Cognitively relevant dynamic high-order correlations in
 \begin{figure}[tp]
   \centering
   \includegraphics[width=\textwidth]{figs/decode_level}
-  \caption{\textbf{Across-participant timepoint decoding accuracy varies with
-      correlation order and cognitive engagement.}
-    \textbf{a.~Decoding accuracy as a function of order: PCA.}
-    \textit{Order} ($x$-axis) refers to the maximum order of dynamic
+  \caption{Across-participant timepoint decoding accuracy varies with
+      correlation order and cognitive engagement.
+    a.~Decoding accuracy as a function of order: PCA.
+    Order ($x$-axis) refers to the maximum order of dynamic
```

Comment:
Change to:

``Order'' ($x$-axis) refers to the maximum order of dynamic

Reply:
done